dask/bytes/tests/test_local.py

```diff
@@ -70,6 +70,17 @@ def test_with_urls():
         assert sum(map(len, values)) == len(files)
 
 
+def test_with_paths():
+    pathlib = pytest.importorskip('pathlib')
+    with filetexts(files, mode='b'):
+        url = pathlib.Path('./.test.accounts.*')
+        sample, values = read_bytes(url, blocksize=None)
+        assert sum(map(len, values)) == len(files)
+    with pytest.raises(OSError):
```

Comment:
It uses urlsplit. There is also a special case up front for windows paths (like c:\path\file) - that is all working on strings. We need to know whether the path is string-like or list-like, and whether it has glob characters before then.